main.py

```diff
@@ -37,13 +39,17 @@
                    description=info.description,
                    help_command=commands.MinimalHelpCommand())
 
+bot.qrm = SimpleNamespace()
+bot.qrm.session = aiohttp.ClientSession(headers={'User-Agent': f'discord-qrm2/{info.release}'})
 
 # --- Commands ---
 
+
```

Comment:
Why is that newline not on line 45?
```diff
+\n
\n
# --- Commands ---\n
\n
-\n
```

Reply:
flake8 told me to add it idk

Reply:
flake8 will not check if a newline is before or after a comment